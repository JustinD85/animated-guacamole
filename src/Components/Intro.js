import React, { Component } from 'react';

class Intro extends Component{

  constructor(props) {
    super(props);
    this.state = {
      name:'some Name'
    }
  }

  createPerson() {
    return 'Person';//props: name
  }

  render() {
  

    return 'Room(Person, people?)';
  }
}

export default Intro;