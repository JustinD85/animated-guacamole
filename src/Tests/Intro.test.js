import React, {Component} from 'react';
import {shallow} from 'enzyme';
import Intro from '../Components/Intro'

describe('<Intro/>', ()=>{
    let wrapper;
    beforeEach(()=>{
        wrapper = <Intro />;
    });

    it('should render', ()=>{
        expect(wrapper).toMatchSnapshot();
    });
    


})
