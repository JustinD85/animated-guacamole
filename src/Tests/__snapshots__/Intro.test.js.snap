// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Intro/> should render 1`] = `<Intro />`;
